### PR TITLE
7876 - added unit tests for modal component

### PR DESCRIPTION
**Pull request by brianpetway**
https://federal-spending-transparency.atlassian.net/browse/DA-7876

### CODE REVIEW

**Comment by anjenkin** [comment]
* Looks good.  I noticed that the line coverage for this test is around 40 or so percent.  I think the resize functions could be tested too to increase the coverage.

**Comment by brianpetway** [comment]
Added a test for the handleResize function. Line coverage is above 70% now. Ready to review again.

File: src/components/modal/modal.spec.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    const { getByTestId } = renderComponent({});
+
+    expect(getByTestId('modalChildContainer')).toHaveStyle('max-width: 716.8px');
+    expect(getByTestId('modalChildContainer')).toHaveStyle('max-height: 494.4000000000001px');
+
+    window.innerWidth = 500;
+    window.innerHeight = 500;
+
+    // resize event fires handleResize in the component
+    global.dispatchEvent(new Event('resize'));
```
Comment (anjenkin) [inline src/components/modal/modal.spec.js:73]: ohh this is really cool, a resize event!

File: src/components/modal/modal.spec.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
```
Comment (anjenkin) [inline src/components/modal/modal.spec.js:2]: Is this import necessary?

File: src/components/modal/modal.spec.js (hunk 1 above)
Comment (brianpetway) [reply src/components/modal/modal.spec.js:73]: thank you very much

File: src/components/modal/modal.spec.js (hunk 2 above)
Comment (brianpetway) [reply src/components/modal/modal.spec.js]: No it was not. Thanks.

**Comment by brianpetway** [comment]
Removed unused imports. Ready to review again.

**Review by anjenkin — APPROVED** [review]

**Pull request merged**